apps/openchallenges/edam-etl/src/main.py

```diff
@@ -1,16 +1,75 @@
 """Extract, transform and load EDAM concepts"""
 
+import pandas as pd
+import requests
 from os import getenv
+from typing import Optional
+from typing import NoReturn
 
-# Get config from the environment variables
-oc_db_url = getenv("OC_DB_URL")
-print(f"OC DB URL: {oc_db_url}")
+"""Get config from the environment variables"""
```

Comment:
nit: this doesn't need to be a docstring - it can remain as a comment 😁

Reply:
changed to comment